include/message_filters/message_traits.h

```diff
@@ -85,8 +85,7 @@ template<typename M>
 struct TimeStamp<M, typename std::enable_if<HasHeader<M>::value>::type >
 {
   static rclcpp::Time value(const M& m) {
-    auto stamp = m.header.stamp;
-    return rclcpp::Time(stamp.sec, stamp.nanosec);
+    return rclcpp::Time(m.header.stamp);
```

Comment:
Nitpicking, but if we not want to rely on the default constructor, we could use:
```suggestion
    return rclcpp::Time(m.header.stamp, RCL_ROS_TIME);
```
Also makes it clearer to understand what time is being used.
https://docs.ros2.org/galactic/api/rclcpp/classrclcpp_1_1Time.html#a13b8e8297e89516806b18c26747804d5